2.1.8

Mariusz89B opened the pull request:
2021.08.1 v.2.1.8
+ Fixed html import error.

2021.08.11 v.2.1.7
+ Fixed M3U playlist generator.

2021.08.09 v.2.1.6
+ Added M3U playlist generator.

2021.08.08 v.2.1.5
+ Added setting "Hide previously aired Live-Tv programmes".

2021.08.07 v.2.1.4
+ Added watched and purchased categories for viaplay.pl.
+ Fixed category error.

2021.08.05 v.2.1.3
+ Added viaplay.pl.
+ Added Polish translation.
+ Added Swedish translation.

Commit:
2.1.3

Commit:
Merge pull request #1 from Mariusz89B/2.1.3

2.1.3

Commit:
2.1.3

Commit:
Create FUNDING.yml

Commit:
2.1.4

Commit:
Merge branch 'master' of https://github.com/Mariusz89B/kodi-viaplay

Commit:
2.1.4

Commit:
2.1.5

Commit:
2.1.6

Commit:
2.1.7

Commit:
2.1.7

Commit:
2.1.7

emilsvennesson commented on .github/FUNDING.yml:
```diff
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
```
Can you remove this file?

Mariusz89B replied:
Should be deleted now.

emilsvennesson commented:
Thanks for your PR!

Personally, I would prefer if we dropped support for Kodi versions < 19 because it adds a lot of if/else logic which is a hassle to maintain in the long run.

Commit:
Delete FUNDING.yml

Mariusz89B reviewed (commented):
> Thanks for your PR!
> 
> Personally, I would prefer if we dropped support for Kodi versions < 19 because it adds a lot of if/else logic which is a hassle to maintain in the long run.

Yes, I know this is a issue, but while Kodi v18 still is mainstream I would like to keep these changes and later just add python dependency to phase out old Kodi versions.

Commit:
2.1.8

Commit:
Merge branch 'master' of https://github.com/Mariusz89B/kodi-viaplay

Commit:
2.1.8

Commit:
2.1.9

Commit:
2.2.0

Commit:
2.2.1

Commit:
2.2.2

Commit:
2.2.3

Commit:
2.2.4

Commit:
2.2.5

Commit:
2.2.5

Commit:
2.2.6

Commit:
2.2.7

Commit:
2.2.7

Commit:
2.2.8

Commit:
2.2.9

Commit:
2.3.0

Commit:
2.3.0

Beanow committed:
Add NL country support

Using viaplay.com domain, but NL country codes.
While viaplay.nl exists, not all APIs work on this domain and mostly
it's used for redirecting to viaplay.com/nl.

Commit:
Merge pull request #3 from Beanow/add-nl

Add NL country support

heppix committed:
Fixed popular sport tiles

zuzia-dev committed:
Merge pull request #4 from zuzia-dev/master

Fixed popular sport tiles

heppix committed:
2.3.1

heppix committed:
Fixed sports series event listing

heppix committed:
2.3.2

zuzia-dev committed:
Merge pull request #5 from zuzia-dev/master

Fixed sport series event listing

heppix committed:
Fixed live tv channels and popular sports in .se

zuzia-dev committed:
Addon: Update version to 2.3.3

zuzia-dev committed:
Bump version to 2.3.3

zuzia-dev committed:
Merge pull request #7 from zuzia-dev/master

Fixed live tv channels and popular sports in .se

Commit:
2.3.4

Commit:
2.3.5

Mariusz89B committed:
2.3.6

Mariusz89B committed:
v2.3.7

Mariusz89B committed:
Update README.md

Mariusz89B committed:
Update README.md

Mariusz89B committed:
Update README.md

Mariusz89B committed:
Update README.md

Mariusz89B committed:
v2.3.8

DenDyGH committed:
v2.3.9

Fix live

Mariusz89B committed:
Merge pull request #11 from DenDyGH/master

v2.3.9

Mariusz89B committed:
v.2.3.9

Mariusz89B committed:
v2.3.9

Mariusz89B committed:
v2.4.0

Mariusz89B committed:
v2.4.0

Mariusz89B committed:
Update README.md

Mariusz89B committed:
Update README.md

Mariusz89B committed:
v2.4.1_v1

Mariusz89B committed:
Merge branch 'master' of https://github.com/Mariusz89B/plugin.video.viaplay

Mariusz89B committed:
v2.4.1_v2

Mariusz89B committed:
v2.4.1_v3

Mariusz89B committed:
v2.4.1_v4

Mariusz89B committed:
v2.4.1_v5

Mariusz89B committed:
v2.4.1_v6

Mariusz89B committed:
v2.4.1_v7

Mariusz89B committed:
v2.4.1_v8

Mariusz89B committed:
v2.4.1

Mariusz89B committed:
v2.4.1

Mariusz89B committed:
v2.4.2

Mariusz89B committed:
v2.4.3_v1

Mariusz89B committed:
v2.4.3_v2

Mariusz89B committed:
v2.4.3_v3

Mariusz89B committed:
v2.4.3_v4

Mariusz89B committed:
v2.4.3

Mariusz89B committed:
v2.4.4

Mariusz89B committed:
v2.4.5_v1

Mariusz89B committed:
v2.4.5_v1

Mariusz89B committed:
v2.4.5_v2

Mariusz89B committed:
v2.4.5

Mariusz89B committed:
v2.4.6_v1

Mariusz89B committed:
v2.4.6_v2

Mariusz89B committed:
v2.4.6_v3
